obal/data/modules/rpmspec_changelog_check.py

```diff
@@ -10,16 +10,11 @@
 from ansible.module_utils.obal import get_changelog_evr, get_specfile_evr  # pylint:disable=import-error,no-name-in-module
 
 
-ANSIBLE_METADATA = {
```

Comment:
What is the reason to remove this?

Reply:
It doesn't seem like it's used anywhere, and pylint was failing it on a `duplicate-code` error that I was unable to disable.

Reply:
As long as this module is not part of ansible proper or we're trying to generate docs using the ansible sphinx (as we do for f-a-m), this data should have no use.

Simple test, if `ansible-doc rpmspec_changelog_check` still works after this change, we're good :)

Reply:
```
$ ansible-doc -M ./ rpmspec_changelog_check
ERROR! module rpmspec_changelog_check missing documentation (or could not parse documentation): 'NoneType' object has no attribute 'get'
```